### PR TITLE
[gdb] Update .deb version and doc improvements

### PR DESCRIPTION
**Pull request by cbiesinger**
Mention how to upgrade and the date of the last rebuild.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ import JsDbg
 end
 ```
 
+### Upgrading
```
Comment (sanketj) [inline docs/README.md]: Perhaps we should call this "Updating".

File: docs/README.md [hunk 2]
```diff
@@ -70,6 +70,14 @@ import JsDbg
 end
 ```
 
+### Upgrading
+
+There is currently no autoupdate on the GDB version of JsDbg, so you have to
+download and install a new version manually to upgrade. Just follow the
```
Comment (sanketj) [inline docs/README.md]: "to upgrade" seems redundant

File: docs/README.md [hunk 3]
```diff
@@ -70,6 +70,14 @@ import JsDbg
 end
 ```
 
+### Upgrading
+
+There is currently no autoupdate on the GDB version of JsDbg, so you have to
+download and install a new version manually to upgrade. Just follow the
+installation instructions again (except for the .gdbinit part).
+
+Last update of the GDB packages: Aug 18th, 2019.
```
Comment (sanketj) [inline docs/README.md]: Aug 12th?

Comment (cbiesinger) [reply docs/README.md]: Er, oops.

All changes made.

**Pull request merged**